From 3dd4d1cdf7cfe296fb8445fa7408deb5daabeb9d Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Mon, 30 Oct 2017 13:22:44 +0800
Subject: [PATCH 064/102] arm64: allwinner: a64: add labels to the SimpleFB
 nodes

Some more properties may be specified for the SimpleFB nodes (e.g.
regulators for LCD panel or the HVCC power pin).

Add a simplefb_{lcd,hdmi} label to the node, in order to allow to
specify the properties.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-a64.dtsi
index b37c538043d2..df1b4467052e 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64.dtsi
@@ -59,7 +59,7 @@
 		#size-cells = <1>;
 		ranges;
 
-		framebuffer-lcd {
+		simplefb_lcd: framebuffer-lcd {
 			compatible = "allwinner,simple-framebuffer",
 				     "simple-framebuffer";
 			allwinner,pipeline = "mixer0-lcd0";
@@ -70,7 +70,7 @@
 			status = "disabled";
 		};
 
-		framebuffer-hdmi {
+		simplefb_hdmi: framebuffer-hdmi {
 			compatible = "allwinner,simple-framebuffer",
 				     "simple-framebuffer";
 			allwinner,pipeline = "mixer1-lcd1-hdmi";
-- 
2.13.6

